libraries/helpers.rb

```diff
@@ -310,6 +310,14 @@ def are_queues_updated?
   config["Scheduling"] != previous_config["Scheduling"] or is_compute_node_bootstrap_timeout_updated?(previous_config, config)
 end
 
+# Verify if CustomSlurmSettings has been updated in the config
```

Comment:
We should have tests for these functions.
They should be added as part of the review of the recipes, but let's try to add them opportunistically.

Let's open a nice-to-have task to add some of them before closing the story.

Reply:
I think this is part of some broader story the QA team is working on.